src/input-login.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Text, TextInput, StyleSheet } from "react-native";
+
+interface InputProps {
+	inputHeader: string;
+	handleTextChange: Function;
```

Comment:
2 pontos aqui:
- funções como parametros a gente costuma utilizar `on`, `handle` a gente utiliza mais pra funções definidas direto no componente, por exemplo
```suggestion
	onTextChange: Function;
```
- `Function` não é recomendado para tipagem de parametros do tipo função pq ela é muito generica e aceita muitas coisas que podem não ser o esperado, o ideal é declarar uma função com os parametros e o retorno corretos. 
Exemplo com função que não recebe parametros e não tem retorno:
```suggestion
	handleTextChange: () => void;
```